src/util/StoreProperty.js

```diff
@@ -0,0 +1,40 @@
+export class StoreProperty {
+	_observer = new Set();
+	_state = {};
+
+	constructor(value) {
+		this._state = { ...value, ...this.properties() };
```

Comment:
@quarkus Right now this assumes that we're always using a properties map (either through the method itself or via the constructor). But I would like to also be able to use Stores/Properties that do not use this at all. For example look at the MouseModel or MatchMediaModel in this branch: https://github.com/webtides/element-js/blob/feature/experimental-decorators/examples/example-element-with-decorators.js
They have their own and more specific constructor parameters. It is doable of course right now. But we would always have to call super with an empty `{}` correct? It's probably fine! But maybe there is an alternative to be discussed here?

Reply:
Can that be solved via primitive Support ?